R/XcmsExperiment.R

```diff
@@ -37,24 +37,19 @@ setMethod("readMsObject",
 #' @noRd
 .store_xcmsexperiment <- function(x, path = tempdir()) {
     .export_process_history(x, path = path)
-    if (xcms::hasChromPeaks(x))
-        .export_chrom_peaks(x, path)
+    ## if (xcms::hasChromPeaks(x)) # maybe also export chromPeaks.
```

Comment:
i'm confused aha. What do you mean maybe ? and the function after is not hashed ? We don't need to do that check ?

Reply:
true - I completely forgot that message. My thought was more what if an user exports an `XcmsExperiment` object without chrom peak results. It would maybe still be nice to be able to import such an "empty" object? so, chromPeaks()` should return ALWAYS a `matrix` with column names (but eventually no rows), so we could export/import that.

I would just like to avoid throwing an error if the `XcmsExperiment` is empty. I don't like errors.

Reply:
I added now unit tests checking export/import of empty objects. and along that I had to fix the export of `MsExperiment`: if `@sampleData` is an empty data frame I'm replacing it with a zero-row data frame with one column (sample_name). Otherwise the `read.table` call will fail.